packages/docs/page-config/ui-elements/dropdown/examples/PlacementAndOffset.vue

```diff
@@ -22,7 +22,7 @@
             :offset="offset"
           >
             <template #anchor>
-              <Coordinates :placement="placement" />
+              <div><Coordinates :placement="placement" /></div>
```

Comment:
Wrappers can be removed now

Reply:
Wrapper is still needed. Without it error is thrown.